tests/clv/models/test_pareto_nbd.py

```diff
@@ -265,6 +356,8 @@ def test_expected_probability_alive(self):
         est_prob_alive_t = self.model.expected_probability_alive(future_t=4.5)
         assert est_prob_alive.mean() > est_prob_alive_t.mean()
 
+        self.covar_model.expected_probability_alive()
```

Comment:
Test something about this. Doesn't the same question of covariates show up here?

Reply:
`xarray` issue needs to be resolved first (see my latest comment).